bot/exts/evergreen/connect_four.py

```diff
@@ -0,0 +1,466 @@
+import asyncio
+import random
+import typing
+from functools import partial
+
+import discord
+import emojis
+from discord.ext import commands
+
+NUMBERS = [
+    ":one:",
+    ":two:",
+    ":three:",
+    ":four:",
+    ":five:",
+    ":six:",
+    ":seven:",
+    ":eight:",
+    ":nine:"
+]
+UNICODE_NUMBERS = [
```

Comment:
Would it be possible to use `bot.constants.Emojis.number_emojis` here instead, so we only need to change it there if we ever need to in future?